src/components/composer/composer/abstract_composer_store.ts

```diff
@@ -55,6 +55,7 @@ export const tokenColors = {
   RIGHT_PAREN: functionColor,
   ARG_SEPARATOR: functionColor,
   MATCHING_PAREN: "#000000",
+  EXTRA_PAREN: "#ff0000",
```

Comment:
nitpick: UNMATCHED_PAREN maybe ?

Reply:
I don't like 'unmatched' because you can have an opening parenthesis while you haven't finished editing your formul --> in which case the parenthesis matches nothing, so you shouldn't highlight it and yet it's not a parenthesis too many

Reply:
ORPHAN_RIGHT_PAREN ?